doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+:::
+
+## Applications:
+
+**Trust:** At a core level, explainability builds trust in the machine learning systems we use. It allows us to justify their use in many contexts where an understanding of the basis of the decision is paramount.
+
+**Testing:** Explainability can be thought of as an extra form of testing for a model. The insights derived should conform to the expected behaviour. Failure to do so may indicate issues with the model or problems with the dataset it's been trained on.
+
+**Functionality:** Insights can be used to augment model functionality. For instance, providing information on top of model predictions such as how to change model inputs to obtain desired outputs.
+
+**Research:** Explainability allows researchers to understand how and why opaque models make decisions. Helping them understand more broadly the effects of the particular model or training schema they're using.
+
+__TODO__:
+- picture of explainability pipeline: training -> prediction -> insight
+
+:::{admonition} **Note 2: Biases**
+
+Practitioners must be wary of using explainability to excuse incorrect models rather than ensuring their correctness. It is possible to have a correctly trained model on a dataset, but the model doesn't reflect reality because the dataset is either wrong or incomplete. Suppose the insights that explainability generates conform to some confirmation bias of the person training the model. In that case, they will be blind to this issue and instead use these methods to confirm erroneous results. A key distinction is that explainability insights are designed to be faithful to the model they are explaining and not the data. You use an explanation to obtain an insight into the data only if the model is trained well.
+:::
+
+## Global and Local Insights
+
+Insights can be categorised into two types. Local and global. Intuitively, a local insight says something about a single prediction that a model makes. For example, given an image classified as a cat by a model, a local insight might give the set of features (pixels) that need to stay the same for that image to remain classified as a cat. Such an insight provides an idea of what the model is looking for when classifying a specific instance into a particular class.
+
+On the other hand, global insights refer to the behaviour of the model over a set of inputs. Plots showing how a regression prediction varies for a given feature while factoring out all the others are examples. These insights provide a more general understanding of the relationship between inputs and model predictions.
+
+__TODO__:
+- Add image to give idea of Global and local insight
+
+## Insights
```

Comment:
I would probably rearrange this section as follows:
- Start with an overview summary table, a little bit like on the [readme](https://github.com/SeldonIO/alibi#model-explanations). This summary table could also be done at the individual section level below. Also describe in very simple but clear terms what questions each method can answer. I believe this is very important as it informs the reader straight away in simple language what method can solve their problem. E.g. use counterfactuals if you are interested which features need to minimally change to flip your prediction, or anchors to understand which features cannot be changed in order to ensure the current prediction. This will often be the level of understanding or the question users go to Alibi with and want an answer to.
- Then start with a section on global explanations
- Then a section on local explanations, separated by type of insight:
   - Feature importance (SHAP & friends, saliency methods such as IG etc)
   - Counterfactuals: prototype, CFRL, Wachter, pertinent negatives (small section on latter)
   - Necessary features to ensure a prediction: anchors, pertinent positives + highlight that they are complementary to counterfactuals ("what do I minimally need to change to flip a prediction" vs. "what should I definitely not change to keep my current prediction")
   - Briefly mention other types of insights which are currently not (yet!) supported by Alibi: similarity based explanations and (interpretable) surrogate models

Each section should really just contain:
- Intuitive explanation of the method, using minimal maths. For more details refer to the designated method page and/or the underlying paper.
- Discuss pros/cons of the method.
- Highlight a use case.
- Add notes on potential pitfalls such as the baselines for IG

Reply:
The summary table could also directly link to the relevant sections such that you don't have to scroll down.